src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -3,266 +3,271 @@
+                border-right: ($clr-datagrid-action-arrow-size - $clr-rem-1px) solid $clr-datagrid-popover-bg-color;
+                border-top: ($clr-datagrid-action-arrow-size - $clr-rem-1px) solid transparent;
+                border-bottom: ($clr-datagrid-action-arrow-size - $clr-rem-1px) solid transparent;
+            }
+
+            .action-item {
+                @include clr-getTypePropertiesForDomElement(dropdown_text, (color, font-size, letter-spacing));
+
+                background: transparent;
+                border: 0;
+                color: $clr-app-font-color-primary;
+                cursor: pointer;
+                display: block;
+                line-height: calc(#{clr-getTypePropertyValueForDomElement(dropdown_text, line-height)} - 1px);
+                margin: 0;
+                padding: 1px 1rem 0;
+                text-align: left;
+                width: 100%;
 
-        .datagrid-head, .datagrid-head .datagrid-row, .datagrid-column {
-            display: block;
-        }
-    }
+                &:hover,
+                &:focus {
+                    text-decoration: none;
+                    background-color: $clr-datagrid-action-popover-hover-color;
+                }
 
-    // Class only applied temporarily when computing columns width
-    .datagrid-computing-columns-width {
-        display: block;
+                &.active {
+                    background: $clr-datagrid-row-hover;
+                    color: $clr-black;
+                }
 
-        .datagrid-body {
-            display: table;
-            table-layout: auto;
-        }
-        .datagrid-head {
-            display: table-header-group;
-        }
-        .datagrid-row {
-            display: table-row-group;
-        }
-        .datagrid-head .datagrid-row {
-            display: table-row;
+                &:focus {
+                    outline: 0;
+                }
+
+                &.disabled,
+                &:disabled {
+                    cursor: not-allowed;
+                    opacity: 0.4;
+                    user-select: none;
+
+                    &:hover {
+                        background: none;
+                    }
+
+                    &:active,
+                    &:focus {
+                        background: none;
+                        box-shadow: none;
+                    }
+                }
+
+                clr-icon {
+                    vertical-align: middle;
+                    transform: translate3d(0px, -1px, 0);
+                }
+            }
         }
-        .datagrid-row-master {
-            display: table-row;
+
+        .datagrid-row-detail-wrapper {
+            flex-basis: 100%;
         }
+
         .datagrid-row-detail {
-            display: none;
-        }
-        .datagrid-column, .datagrid-cell {
-            display: table-cell;
-        }
-        .datagrid-column-separator {
-            display: none;
+            &.datagrid-container {
```

Comment:
Why did we get rid of the `display: none` in the `.datagrid-row-detail` selector?

Reply:
It's still there, because the file was rewritten the classes used for computation are at the bottom of the file.